src/main/java/ch/wisv/areafiftylan/extras/rfid/model/RFIDLink.java

```diff
@@ -19,23 +19,30 @@
 
 import ch.wisv.areafiftylan.exception.InvalidRFIDException;
 import ch.wisv.areafiftylan.products.model.Ticket;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.hibernate.validator.constraints.Length;
 
 import javax.persistence.*;
 
 @Entity
+@Data
+@NoArgsConstructor
 public class RFIDLink {
+
     @Id
     @GeneratedValue
     private Long id;
 
+    @NonNull
+    @Length(min = RFID_CHAR_COUNT, max = RFID_CHAR_COUNT)
```

Comment:
You're right, we don't. The current tests we have are lacking and strange. I will remove this and handle it in a seperate Issue.